### DISCUSSION TITLE
Would love to see future plans to create desktop versions.

### DISCUSSION THREAD

**Discussion by Christopher-Chros** [bromite/bromite#1232] [Ideas]
I really love the Android app and can't wait to see what the future holds. Would love to see the app coming to Linux and Windows. Would be a great replacement for Brave or Firefox, in the privacy-oriented market of browsers. Especially since both Brave and Firefox have been known for doing sketchy things, this would be a refreshing replacement on the desktop just like it is on Android.

**Comment by ArigatoDevs** [comment]
I feel the same. Bromite is the best browser I have ever used. Thanks to devs for making it.
